### ISSUE TITLE
Keyboard Navigation Shortcut Failure on Windows Native Installation

### ISSUE THREAD

**Issue by daspl1ff** [anthropics/claude-code#3651]

**Comment by ant-kurt** [comment]
@daspl1ff which keyboard navigation shortcut are you referring to?

**Comment by CoffeeAndLoot** [comment]
I believe the release notes referred to alt+m to change modes. I am not sure exactly what they are referring to, but it seems like might be it, because I came here looking to see if anyone else is having the same issue of alt + m to change modes and it does not work.

**Comment by ant-kurt** [comment]
v1.0.54 changed "shift+tab" to "alt+m" for native Windows only (see #3368). If you're still seeing this issue on v1.0.54 or are talking about another shortcut, let me know!

**Comment by CoffeeAndLoot** [comment]
Yea, had to force to v1.0.54 and now its working!

**Comment by emberfish** [comment]
Fixed mine by updating Node on my machine to 24.4.1. Now I can do shift+tab on windows native 👌
